From f71c48709e686ed40536c070682f5fc4cd474630 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tiziano=20M=C3=BCller?= <tm@dev-zero.ch>
Date: Thu, 15 Aug 2013 08:10:04 +0200
Subject: [PATCH] fix LDFLAGS position and add -lcap to LIBS instead of LDFLAGS

---
 configure.ac    |  2 +-
 src/Makefile.in | 16 ++++++++--------
 2 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/configure.ac b/configure.ac
index 83aa4bf..32fcaa0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -159,7 +159,7 @@ AC_CHECK_FUNCS(
 	[cap_get_proc],,
 	AC_CHECK_LIB([cap], [cap_get_proc],
 		AC_DEFINE(HAVE_CAP_GET_PROC)
-		LDFLAGS="$LDFLAGS -lcap",
+		LIBS="$LIBS -lcap",
 	)
 )
 
diff --git a/src/Makefile.in b/src/Makefile.in
index 12591d0..904b75f 100644
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -67,22 +67,22 @@ OBJS = $(SRCS:.c=.o)
 all: ${BINARIES}
 
 jk_chrootlaunch: jk_chrootlaunch.o jk_lib.o utils.o
-	$(CC) $(DEFS) -o $@ jk_chrootlaunch.o jk_lib.o utils.o $(LDFLAGS) $(LIBS)
+	$(CC) $(DEFS) $(LDFLAGS) -o $@ jk_chrootlaunch.o jk_lib.o utils.o $(LIBS)
 
 jk_socketd: jk_socketd.o jk_lib.o utils.o iniparser.o
-	$(CC) $(DEFS) -o $@ jk_socketd.o jk_lib.o utils.o iniparser.o $(LDFLAGS) $(LIBS)
+	$(CC) $(DEFS) $(LDFLAGS) -o $@ jk_socketd.o jk_lib.o utils.o iniparser.o $(LIBS)
 
 jk_lsh: jk_lsh.o iniparser.o jk_lib.o utils.o wordexp.o
-	$(CC) $(DEFS) -o $@ jk_lsh.o iniparser.o jk_lib.o utils.o wordexp.o $(LDFLAGS) $(LIBS)
+	$(CC) $(DEFS) $(LDFLAGS) -o $@ jk_lsh.o iniparser.o jk_lib.o utils.o wordexp.o $(LIBS)
 
 jk_chrootsh: jk_chrootsh.o iniparser.o jk_lib.o utils.o passwdparser.o
-	$(CC) $(DEFS) -o $@ jk_chrootsh.o iniparser.o jk_lib.o utils.o passwdparser.o $(LDFLAGS) $(LIBS)
+	$(CC) $(DEFS) $(LDFLAGS) -o $@ jk_chrootsh.o iniparser.o jk_lib.o utils.o passwdparser.o $(LIBS)
 
 jk_uchroot: jk_uchroot.o iniparser.o jk_lib.o utils.o
-	$(CC) $(DEFS) -o $@ jk_uchroot.o iniparser.o jk_lib.o utils.o $(LDFLAGS) $(LIBS)
+	$(CC) $(DEFS) $(LDFLAGS) -o $@ jk_uchroot.o iniparser.o jk_lib.o utils.o $(LIBS)
 
 jk_procmailwrapper: jk_procmailwrapper.o jk_lib.o utils.o
-	$(CC) $(DEFS) -o $@ jk_procmailwrapper.o jk_lib.o utils.o $(LDFLAGS) $(LIBS)
+	$(CC) $(DEFS) $(LDFLAGS) -o $@ jk_procmailwrapper.o jk_lib.o utils.o $(LIBS)
 
 clean:
 	rm -f ${BINARIES}
@@ -114,7 +114,7 @@ uninstall:
 	rm -f  ${prefix}/sbin/jk_procmailwrapper
 
 iniparsertester: iniparser.o iniparsertester.o jk_lib.o
-	$(CC) $(DEFS) -o iniparsertester iniparsertester.o iniparser.o jk_lib.o $(LDFLAGS) $(LIBS)
+	$(CC) $(DEFS) $(LDFLAGS) -o iniparsertester iniparsertester.o iniparser.o jk_lib.o $(LIBS)
 
 passwdparsertester: passwdparser.o passwdparsertester.o
-	$(CC) $(DEFS) -o passwdparsertester passwdparsertester.o passwdparser.o $(LDFLAGS) $(LIBS)
+	$(CC) $(DEFS) $(LDFLAGS) -o passwdparsertester passwdparsertester.o passwdparser.o $(LIBS)
-- 
1.8.2.1

